woocommerce/woocommerce-android
# Fix widget for woo express stores

### Description
This PR adds a fix to allow the stats widget to work when Jetpack is not properly configured. For example: Woo Express stores have jetpack stats disabled by default. It does that by making the visitor stats optional and using `null` when the visitor stats fail to load.

### Testing instructions
TC1
1. Use a site with a Jetpack connection
2. Complete a new order using the app
3. Add the Today's stats widget
4. Check that the Revenue, Orders and Visitors stats are displaying the correct data

TC2
1. Create a new trial store
2. Complete a new order using the app
3. Add the Today's stats widget
4. Check that the Revenue, Orders, Visitors stats are displaying the correct data
5. Check that Visitors stats is not displayed

### Images/gif
<img width="300" src="https://github.com/woocommerce/woocommerce-android/assets/18119390/97f8f27a-dabb-4092-985e-94cfad81d5f9" />
